content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -0,0 +1,296 @@
+---
+reviewers:
+- liggitt
+- jpbetz
+- cici37
+title: Validating Admission Policy
+content_type: concept
+---
+
+<!-- overview -->
+
+{{< feature-state state="alpha" for_k8s_version="v1.26" >}}
+
+This page provides an overview of Validating Admission Policy.
+
+
+<!-- body -->
+
+## What is Validating Admission Policy?
+
+Validating admission policies offer a declarative, in-process alternative to validating admission webhooks.
+
+Validating admission policies use the Common Expression Language (CEL) to declare the validation rules of a policy. 
+Validation admission policies are highly configurable, enabling policy authors to define policies that can be parameterized and scoped to resources as needed by cluster administrators.
+
+## What Resources Make a Policy
+
+A policy is generally made up of three resources:
+
+- The `ValidatingAdmissionPolicy` describes the abstract logic of a policy (think: "this policy makes sure a particular label is set to a particular value").
+
+- A `ValidatingAdmissionPolicyBinding` links the above resources together and provides scoping. If you only want to require an `owner` label to be set for `Pods`, the binding is where you would specify this restriction.
+
+- A parameter resource provides information to a ValidatingAdmissionPolicy to make it a concrete statement (think "the `owner` label must be set to something that ends in `.company.com`"). A native type such as ConfigMap or a CRD defines the schema of a parameter resource. `ValidatingAdmissionPolicy` objects specify what Kind they are expecting for their parameter resource.
+
+
+At least a `ValidatingAdmissionPolicy` and a corresponding  `ValidatingAdmissionPolicyBinding`  must be defined for a policy to have an effect.
+
+If a `ValidatingAdmissionPolicy` does not need to be configured via parameters, simply leave `spec.paramKind` in  `ValidatingAdmissionPolicy` unset.
+
+## Getting Started with Validating Admission Policy
+
+Validating Admission Policy is part of the cluster control-plane. You should write and deploy them with great caution. In the following, we describe how to quickly experiment with Validating Admission Policy.
+
+Prerequisites
+- Ensure the `ValidatingAdmissionPolicy` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled.
+- Ensure that the admissionregistration.k8s.io/v1alpha1 API is enabled.
+- Make sure the user has access to read the resources which are validated on.
```

Comment:
There is a special case. For a non-nil ParamKind in ValidatingAdmissionPolicy, we need the user to have read access to all of the given resource (resolvable) or all resources of the group (unresolvable). Would you like to mention it?

Reply:
Thanks! Added as a separate section